agent/lib/kontena/workers/stats_worker.rb

```diff
@@ -117,7 +118,7 @@ def send_container_stats(container)
         }
       }
       self.queue << event
-      send_statsd_metrics(container.dig(slice_key, :aliases, 0), event[:data])
+      send_statsd_metrics(container.dig(:aliases, 0), event[:data])
```

Comment:
Changed to use `name = container[:aliases].select{|a| a != container[:id]}.first`